application/CohortManager/src/Functions/ExceptionHandling/CreateException/CreateException.cs

```
@@ -36,15 +36,20 @@ public async Task<HttpResponseData> RunAsync([HttpTrigger(AuthorizationLevel.Ano
                 requestBody = await reader.ReadToEndAsync();
                 exception = JsonSerializer.Deserialize<ValidationException>(requestBody);
             }
-            if (!string.IsNullOrEmpty(requestBody))
+
+            if (string.IsNullOrEmpty(requestBody))
+            {
+                _logger.LogError("The requestBody is empty, unable to create exception record");
+                return _createResponse.CreateHttpResponse(HttpStatusCode.BadRequest, req);
+            }
+
+            if (_validationData.Create(exception))
             {
-                if (_validationData.Create(exception))
-                {
-                    return _createResponse.CreateHttpResponse(HttpStatusCode.OK, req);
-                }
+                _logger.LogInformation("The exception has been created successfully");
+                return _createResponse.CreateHttpResponse(HttpStatusCode.Created, req);
             }
-            return req.CreateResponse(HttpStatusCode.BadRequest);
 
+            return _createResponse.CreateHttpResponse(HttpStatusCode.OK, req);
```

If this didn't create an exception we should return an error of some sort?

---

I've updated this to log an error message with the exception. It also now returns an internal server error. 

I'm not sure if this is the best status code to use though. What do you think @MWClayson-NHS?